rules/opentitan.bzl

```diff
@@ -767,15 +851,19 @@ def opentitan_functest(
         )
 
     if "cw310" in targets:
+        # Set default CW310 FPGA parameters if none are provided.
+        if cw310 == None:
+            cw310 = cw310_params()
+
         if test_in_rom:
-            fail("test_in_rom only valid on Verilator target.")
+            fail("test_in_rom only valid on DV and Verilator targets.")
```

Comment:
Can we cut down on future work keeping this accurate by saying "not valid in cw310 targets?

Reply:
Fixed. Changed to "only valid on simulation targets". Technically we could make these type of tests work on FPGA too, but that would require splicing and thats not automated in the bazel flow, so kept to "simulation targets".